server/controllers/DocumentControllers.js

```diff
@@ -0,0 +1,215 @@
+import pagination from '../helper/pagination';
+
+const Document = require('../models').Document;
+const User = require('../models').User;
+
+const DocumentControllers = {
+  /**
+   * @param {Object} req
+   * @param {Object} res
+   * @returns {Object} returns a response object.
+   */
+  create(req, res) {
+    const authorId = res.locals.decoded.UserId;
+    return Document.create({
+      title: req.body.title,
+      content: req.body.content,
+      UserId: authorId,
+      access: req.body.access
+    })
+    .then(document => res.status(201).send({
```

Comment:
indentation here is funky. move the opening brace into a line of it's own and indent `document...` inside it